doc/source/whatsnew/v0.25.2.rst

```diff
@@ -101,6 +101,7 @@ Other
 ^^^^^
 
 - Compatibility with Python 3.8 in :meth:`DataFrame.query` (:issue:`27261`)
+- Backtick quoting in :meth:`DataFrame.query` and :meth:`DataFrame.eval` can now also be used to use invalid identifiers like names that start with a digit, are python keywords, or are using single character operators.
```

Comment:
Can you move this to the 1.0.0 whatsnew?

Reply:
OK